Sources/NIO/CircularBuffer.swift

```diff
@@ -682,6 +719,20 @@ extension CircularBuffer: RangeReplaceableCollection {
         }
         return self[self.startIndex]
     }
+
+    /// Prepares the `CircularBuffer` to store the specified number of elements.
+    @inlinable
+    public mutating func reserveCapacity(_ capacity: Int) {
```

Comment:
Should we make it clearer that we'll reserve enough capacity for _at least_ the specified number of elements and use `minimumCapacity` rather than `capacity` for the label?

Reply:
Strictly the label is `_`. We can’t change that because the label is a part of the protocol requirement. We can change the inner name to `minimumCapacity`, but I’m not sure if that helps at all.

Reply:
The inner label shows up in autocompletion which I find helpful. We could also just mention in the docs.